Hi!

I was having the same issue on both Windows 10 and linux containers:

```
[ERROR] Failed to execute goal ro.isdc.wro4j:wro4j-maven-plugin:1.7.9:run (default) on project hadoop-yarn-ui: Execution default of goal ro.isdc.wro4j:wro4j-maven-plugin:1.7.9:run failed: An API incompatibility was encountered while executing ro.isdc.wro4j:wro4j-maven-plugin:1.7.9:run: java.lang.ExceptionInInitializerError: null
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>ro.isdc.wro4j:wro4j-maven-plugin:1.7.9
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/nicolas/.m2/repository/ro/isdc/wro4j/wro4j-maven-plugin/1.7.9/wro4j-maven-plugin-1.7.9.jar
```
(truncated a part of the stacktrace)
```
[ERROR] urls[99] = file:/home/nicolas/.m2/repository/log4j/log4j/1.2.17/log4j-1.2.17.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>org.apache.hadoop:hadoop-project:3.5.0-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] : NullPointerException
```

I was able to test the fix locally and it worked for me. Thanks a lot!

--- a/LICENSE-binary
+++ b/LICENSE-binary
@@ -362,6 +362,7 @@ org.objenesis:objenesis:2.6
 org.xerial.snappy:snappy-java:1.1.10.4
 org.yaml:snakeyaml:2.0
 org.wildfly.openssl:wildfly-openssl:2.1.4.Final
+ro.isdc.wro4j:wro4j-maven-plugin:1.8.0
 software.amazon.awssdk:bundle:2.25.53
 net.jodah:failsafe:2.4.4
 

